python-sdk/tests/benchmark/Makefile

```diff
@@ -37,6 +38,7 @@ container: clean
 	@echo "Building and pushing container $(CONTAINER_REGISTRY)"
 	@gcloud auth configure-docker gcr.io
 	@docker build --build-arg=GIT_HASH=$(GIT_HASH) --tag=$(CONTAINER_REGISTRY):$(GIT_HASH) --tag=$(CONTAINER_REGISTRY):latest -f ./Dockerfile  ../../
+	@gcloud auth activate-service-account --key-file $(GOOGLE_APPLICATION_CREDENTIALS)
```

Comment:
Since we are introducing this dependency on the environment variable, it may be worth to make this dependency explicit in line 37, by adding `check_google_credentials`:
```
container: clean check_google_credentials
```